apps/tasks.py

```diff
@@ -649,3 +503,158 @@ def delete_old_objects():
     old_apps = AppInstance.objects.filter(created_on__lt=threshold_time, app__category__name="Develop")
     for app_ in old_apps:
         delete_resource.delay(app_.pk)
+
+
+@shared_task(bind=True, max_retries=None)
+def init_event_listener(self, namespace, label_selector):
+    """
+    The event listener takes the latest event and checks if a corresponding appinstance
+    should be updated. It uses the creation timestamp to always use the status of the youngest pod
+    in a helm release.
+    """
+    api, w = setup_client()
+    try:
+        my_dict = {}
+        for event in w.stream(api.list_namespaced_pod, namespace=namespace, label_selector=label_selector):
+            pod = event["object"]
+
+            status = get_status(pod)
```

Comment:
```suggestion
            status: str = get_status(pod)
```

This would make it easier to read :)

Reply:
I agree! We should do more of this generally